### PR TITLE
Speedometer: Handle lite mode popup

### PR DESCRIPTION
**Pull request by scojac01**
If Chrome has been unused before then the lite mode dialog
will appear. This improvement handles that specific case.

### CODE REVIEW

File: wa/workloads/speedometer/uiauto/app/src/main/java/com/arm/wa/uiauto/speedometer/UiAutomation.java [hunk 1]
```diff
@@ -47,11 +47,11 @@ public void initialize(){
     @Test
     public void setup() throws Exception{
         setScreenOrientation(ScreenOrientation.NATURAL);
-        clearFirstRun();
     }
 
     @Test
     public void runWorkload() throws Exception {
+        clearFirstRun();
```
[the remark this answers is not in the document]
Comment (scojac01) [reply wa/workloads/speedometer/uiauto/app/src/main/java/com/arm/wa/uiauto/speedometer/UiAutomation.java]: I moved the clear first run as it didn't seem to trigger in the setup stage on older devices.

**Comment by scojac01** [comment]
I haven't yet tested MotionMark but I imagine it will also be the same yes. I can implement it in that workload too.

File: wa/workloads/speedometer/uiauto/app/src/main/java/com/arm/wa/uiauto/speedometer/UiAutomation.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcbonnici) [reply wa/workloads/speedometer/uiauto/app/src/main/java/com/arm/wa/uiauto/speedometer/UiAutomation.java]: Do you mean all of the popup clearing or just this new one? 
We tend to try and keep as little as extra steps in the run stage of the workload as possible.

**Comment by marcbonnici** [comment]
> I haven't yet tested MotionMark but I imagine it will also be the same yes. I can implement it in that workload too.

If this is going to be common to all web based workloads then it might be worth adding a function to dismiss chrome popups to the UiAutomator base class to prevent having to maintain then in the separate workloads?

File: wa/workloads/speedometer/uiauto/app/src/main/java/com/arm/wa/uiauto/speedometer/UiAutomation.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (scojac01) [reply wa/workloads/speedometer/uiauto/app/src/main/java/com/arm/wa/uiauto/speedometer/UiAutomation.java]: I'll retest this and see what does/doesn't work.

**Pull request merged**